examples/http/client.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      gRPC://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require('path');
+const http = require('http');
+const tracing = require('@opencensus/nodejs');
+const { plugin } = require('@opencensus/instrumentation-http');
+const { ZipkinTraceExporter } = require('@opencensus/exporter-zipkin');
+const { TraceContextFormat } = require('@opencensus/propagation-tracecontext');
+
+const tracer = setupTracerAndExporters();
+
+/** A function which makes requests and handles response. */
+function makeRequest () {
+  tracer.startRootSpan({ name: 'octutorialsClient.makeRequest' }, rootSpan => {
```

Comment:
On first reading, it looked to me like the `tracer.startRootSpan` was starting a span for the outgoing request, but really it's a parent span for it.

Could you add a comment that explains what the root span is here and that it will get a child span for the HTTP request?

Reply:
Added, PTAL

Reply:
:+1: